README.md

```diff
@@ -7,61 +7,198 @@
-} else if temperatureInFahrenheit >= 90 {
-    print("Very hot! ☀️")
+if rainingOutside == true {
+	takeUmbrella()
 } else {
-    print("Just right. 🌻")
+	// don't worry about taking the umbrella
+	// NB no code is being excuted here, only comments!
+}
+goForAWalk()
+````
+is the same as...
+
+````Swift
+func takeUmbrella() {
+    // grab umbrella
+    print("I have my umbrella! ☂️")
 }
-```
 
 
+}
+goForAWalk()
+````
 
-## What the student can do at this point 
+Our ````if```` doesn't have to be followed by a Boolean expression; it can also take Boolean variable or constant, or a function which returns a Boolean.  Here's what that would look like in code:
 
-* Create variables and constants
-* Is familiar with type annotations, type inference and string interpolation.
-* Can create functions with return types.
-* Is familiar with the `String`, `Int`, `Double` and `Bool` type.
-* Just learned about `Bool` prior to this reading.
-* Just learned about *comparison operators* but has YET to see what an if statement looks like. 
+````Swift
+let x = true
+if x {
+	...
+}
+````
 
+is the same as
 
-## Outline / Notes
+````Swift
+let x = true
+if x == true {
+	...
+}
+````
+
+And we can use variables:
+
+````Swift
+var y = false
+if y {
+...
+}
+````
+
+Which is the same as:
+
+````Swift
+var y = false
+if y == true {
+   ...
+ }
+ ````
+ 
+And here we'll show how to use a function:
+
+````Swift
+func test(Bool a, Bool b) -> Bool {
+...
+}
+
+if test(x, y) {
+...
+}
+````
+
+There is one final bit of magic with Conditionals, and that is that we can chain them together almost indefinitely.  This is something that's actually much clearer to understand in code than written out in English, so let's look at an example:
+
+````Swift
+if sallyDressColor == "yellow" {
+	wearSomethingGreen()
+} else if sallyDressColor == "blue" {
+	wearSomethingPurple()
+} else if sallyDressColor == "red" {
+	wearSomethingOrangle() 
+} else if sallyDressColor == "green" {
+	wearSomethingBlue()
+} else ...
+// that's a lot of decision-making!
+````
+
+##Review
+*You've learned that Boolean expressions return ````true```` or ````false```` based upon the result of the comparison performed (or inverting the value in the case of the spcial operator ````!````).
+
+*Boolean expressions are particularly useful in helping your program take a particular course of action.  For example, ````if the battery is low then alert user to conserve power````.  This is expressed with an ````if```` conditional.
+
+*An alternate course of action can also be specified, such as ````if the battery is low then set battery color to red otherwise set battery color to blue````.   You accomplish this using an ````if / else```` Conditional, such as ````if batteryIsLow { ... } else { ... }````
+
+*You can chain multiple ````if / else```` chunks together to handle more complicated situations where there instead of testing a single value you need to take different actions based upon three or more values.
```

Comment:
Simple formatting fix on these four points here below **Review**. You just need to add a space after the asterisk so it becomes a bullet point.